R/commoncor.R

```diff
@@ -0,0 +1,182 @@
+.getPairsIndeces <- function(options) {
```

Comment:
I thought that these functions were already in commoncor, what happened here? Are they removed from the original files?

Reply:
That's the whole `commoncor.R` file copied to this module, as it was not put here initially (i.e., without it bayesian correlations break)